src/main/flight/failsafe.h

```diff
@@ -129,9 +129,10 @@ typedef enum {
 
 typedef struct failsafeState_s {
     int16_t events;
-    bool monitoring;
-    bool active;
-    bool controlling;
+    bool monitoring;                        // Flag that failsafe is monitoring RC link
+    bool suspended;                         // Failsafe is temporary suspended
+    bool active;                            // Failsafe is active (on RC link loss)
+    bool controlling;                       // Failsafe is driving the sticks instead of pilot
```

Comment:
I appreciate the comments here. We're really missing comments in a bunch of places what would help potential new developers a lot. Could you also add under which conditions `suspended` could be true?